lib/Cake/Test/Case/Utility/CakeTextTest.php

```diff
@@ -16,18 +16,18 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
-App::uses('String', 'Utility');
+App::uses('CakeText', 'Utility');
```

Comment:
Seems reasonable, safe and consistent with other class name conventions we have. I know the few places we didn't use Cake prefixes have repeatedly caused issues (File, Folder).